src/main/java/com/iridium/iridiumskyblock/Reward.java

```diff
@@ -0,0 +1,29 @@
+package com.iridium.iridiumskyblock;
+
+import com.cryptomorin.xseries.XSound;
+import com.iridium.iridiumskyblock.database.Island;
+import lombok.NoArgsConstructor;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+
+@NoArgsConstructor
+public class Reward {
+    public Item item;
+    public List<String> commands;
+    public int islandExperience;
+    public XSound sound;
```

Comment:
Why do you have a sound for every reward if you only use the one of the mission?

Reply:
Rewards havent been implemented yet, alot of people wanted commands as rewards so to do this ima have a /is rewards gui the sound is going to be the sound that plays where you collect the rewards from that GUI